### PR TITLE
fix about us page styling

### LINKED ISSUE #155: About Us Page Styling
The original page has the content wrapped in a color like this: 
image

Ours has the color continue past the content to the bottom of the container like this:

image

### PR DESCRIPTION
**Pull request by samuelopra**
Fix #155 

Test URLs:
- Before: https://main--hubblehomes-com--aemsites.hlx.live/about-us
- After: https://issue-155--hubblehomes-com--aemsites.hlx.live/about-us

### CODE REVIEW

File: blocks/columns/columns.css [hunk 1]
```diff
@@ -110,9 +109,10 @@
     }
 
     @media (width >= 900px) {
-      flex: 1;
+      flex: 1 0 auto;
       margin-bottom: 0;
       text-align: left;
+      max-width: calc(33.33% - 21.33px);
```
Comment (bhellema) [inline blocks/columns/columns.css:115]: splitting pixels!

**Review by bhellema — APPROVED** [review]

**Comment by codevoyage** [comment]
I think the title “About Us” should be centred to match the other page headers, for example:

Screenshot 2024-07-10 at 10 02 45 AM

**Review by codevoyage — COMMENTED** [review]
I think the title of the “About Us” page should be centered to match the other page headers.
For example:
Screenshot 2024-07-10 at 10 02 45 AM

**Comment by samuelopra** [comment]
> I think the title of the “About Us” page should be centered to match the other page headers. For example: Screenshot 2024-07-10 at 10 02 45 AM

hey @codevoyage im not sure whether @dave-fink intentionally left-aligned the content so something to bring up in the next meeting, so we can create a task if it's not intentional. For now, ill merge this

**Pull request merged**